src/kotlin/org/intellij/plugins/hcl/formatter/HCLFormattingBuilderModel.kt

```diff
@@ -44,6 +44,7 @@ open class HCLFormattingBuilderModel(val language: Language = HCLLanguage) : For
         .after(EQUALS).spacing(spacesAroundAssignment, spacesAroundAssignment, 0, false, 0)
         .afterInside(IDENTIFIER, BLOCK).spaces(1)
         .afterInside(STRING_LITERAL, BLOCK).spaces(1)
+        .between(L_CURLY, R_CURLY).none()
```

Comment:
Actually this change contradicts with one on next line and one four lines below.
Though I'll investigate that a bit later